src/cpp/server/server.cc

```diff
@@ -286,7 +285,9 @@ bool Server::Start() {
   if (!has_generic_service_) {
     unknown_method_.reset(new RpcServiceMethod(
         "unknown", RpcMethod::BIDI_STREAMING, new UnknownMethodHandler));
-    sync_methods_->emplace_back(unknown_method_.get(), nullptr);
+    // Use of emplace_back with just constructor arguments is not accepted
+    // by gcc-4.4 because nullptr is an anonymous class, so we're constructing 
```

Comment:
Hmm, I thought this seemed finished, but looking again, I can see why it might seem unfinished. I can expand. Need to avoid using an implicit object on a transitive verb, I guess.